gui/wxpython/lmgr/statusbar.py

```diff
@@ -0,0 +1,103 @@
+"""
+@package frame.statusbar
+
+@brief Classes for main window statusbar management
+
+Classes:
+ - statusbar::SbMask
+
+(C) 2006-2011 by the GRASS Development Team
+
+This program is free software under the GNU General Public License
+(>=v2). Read the file COPYING that comes with GRASS for details.
+
+@author Vaclav Petras <wenzeslaus gmail.com>
+@author Anna Kratochvilova <kratochanna gmail.com>
+@author Linda Kladivova <lindakladivova gmail.com>
+"""
+
+import wx
+
+from core.gcmd import RunCommand
+from gui_core.wrap import Button
+from grass.script import core as grass
+
+
+class SbMask:
+    """Button to show whether mask is activated and remove mask with
+    left mouse click
+    """
+
+    def __init__(self, parent, giface):
+        self.name = "mask"
+        self.mask_layer = "MASK"
+        self.parent = parent
+        self.giface = giface
+        self.widget = Button(
+            parent=parent, id=wx.ID_ANY, label=_(self.mask_layer), style=wx.NO_BORDER
+        )
+        self.widget.Bind(wx.EVT_BUTTON, self.OnRemoveMask)
+        self.widget.SetForegroundColour(wx.Colour(255, 0, 0))
+        self.widget.SetToolTip(tip=_("Left mouse click to remove the MASK"))
+        self.giface.updateMap.connect(self.Refresh)
```

Comment:
I think this connect can be removed now, because you added the grassdbChanged signal in gconsole. But please test.